packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -2972,6 +2972,7 @@ export class WalletService {
     // One fee per TXID
     const indexedFee: any = _.keyBy(_.filter(txs, { category: 'fee' } as any), 'txid');
     const indexedSend = _.keyBy(_.filter(txs, { category: 'send' } as any), 'txid');
+    var txWithMisunderstoodMoves = [];
```

Comment:
better use `let`

Reply:
Nice! I'll change to _let_ and try the use of _const_.